### PR TITLE
feat(eden): load existing eden rate into eden scope

### PR DESCRIPTION
**Pull request by leisterfrancisco**
### GH Issue

Load existing eden members on default scope into new eden scope once the smart contract is updated

### CODE REVIEW

File: contracts/rateproducer/src/rateproducer.cpp [hunk 1]
```diff
@@ -491,6 +491,36 @@ namespace eoscostarica {
...
+                row.user = itr->user;
+                row.bp = itr->bp;
+                row.transparency = itr->transparency;
+                row.infrastructure = itr->infrastructure;
+                row.trustiness = itr->trustiness;
+                row.community = itr->community;
+                row.development = itr->development ;   
+            });
+            //save stats
+            save_bp_stats(eden_scope,
```
Comment (xavier506) [inline contracts/rateproducer/src/rateproducer.cpp:512]: where are we checking if the user is an eden member?

Comment (xavier506) [reply contracts/rateproducer/src/rateproducer.cpp:512]: @leisterfrancisco

Comment (leisterfrancisco) [reply contracts/rateproducer/src/rateproducer.cpp:512]: you right, I'm gonna make that last change with the migration too

Comment (leisterfrancisco) [reply contracts/rateproducer/src/rateproducer.cpp:512]: thanks so much for your observation

**Review by xavier506 — APPROVED** [review]
LGTM

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**